src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -157,71 +157,48 @@ ColumnLayout {
     }
 
     RowLayout {
-        id: relayUpsell
-        objectName: _objectName + "-relayUpsell-layout"
+        id: upgradeToAnnualSub
+        objectName: _objectName + "-upgradeToAnnualSub-layout"
 
         visible: false
         Layout.fillWidth: true
         Layout.topMargin: MZTheme.theme.listSpacing
         Layout.bottomMargin: MZTheme.theme.windowMargin
 
-        Component.onCompleted: {
-            if (visible) {
-                Glean.sample.bundleUpsellViewed.record();
-            }
-        }
-
-        ColumnLayout {
+        MZBoldInterLabel {
+            color: MZTheme.colors.purple70
+            horizontalAlignment: Text.AlignLeft
```

Comment:
`horizontalAlignment` of `MZBoldInterLabel` is `Text.AlignLeft` by default, no need to re-bind